cloud/client.go

```diff
@@ -900,29 +1095,67 @@ func (c *client) loadAuthToken() error {
 	if err != nil {
 		return errors.Wrap(err, "failed to read file")
 	}
+	parts := strings.SplitN(string(data), " ", 2)
+	c.authToken = parts[0]
+	c.authTokenExpiry, err = time.Parse(tokenExpiryLayout, parts[1])
+	if err != nil {
+		return errors.Wrapf(err, "unable to parse token expiry: %s", parts[1])
+	}
+	return nil
+}
+
+func (c *client) loadCredentials() error {
+	credPath, err := c.getCredentialsPath(false)
+	if err != nil {
+		return err
+	}
+	if !fileutil.FileExists(credPath) {
```

Comment:
Is it possible to do this after the `ReadFile` via `errors.Is(err, os.ErrNotExist)`?